### ISSUE TITLE
Update Windows nodes to Visual Studio 2017

### ISSUE THREAD

**Issue by clalancette** [ros2/build_farmer#69]
It would be nice to upgrade the Windows nodes to use Visual Studio 2017.  Opening this issue to keep it in mind.  Doing this is currently blocked on Connext https://github.com/ros2/build_cop/issues/61#issuecomment-337955537

**Comment by dirk-thomas** [comment]
We can request a Visual Studio 2017 build of Connext anytime if that is the only reason holding us back.

**Comment by mikaelarguedas** [comment]
Users cannot download such build from the connext website so I don't think it's desired to diverge from users setup.
Do we have a list of serious advantages that this migration would bring? We could then evaluate if it justifies diverging from users setup

**Comment by dirk-thomas** [comment]
> Users cannot download such build from the connext website so I don't think it's desired to diverge from users setup.

I wouldn't state that without confirmation from RTI first if or when they would publish this publically.

**Comment by mikaelarguedas** [comment]
It is accurate to say that currently we and other regular users cannot download it. I'm not making any assumption on RTI's timeline but will be happy to switch when/if this becomes available on their website.

**Comment by dirk-thomas** [comment]
RTI has already confirmed in past conversations that they do have VS2017 builds. So I think it would be helpful to take initiative and ask them explicitly to make them available rather then just to wait until that happens on its own (since that might only be for Connext 6.4).

**Comment by mikaelarguedas** [comment]
:+1: let's do that then

**Comment by dirk-thomas** [comment]
Closing this in favor of ros2/ci#122.
